spring-boot-task/src/main/java/com/example/task5/repository/DBDataStore.java

```diff
@@ -0,0 +1,246 @@
+                        rs.getString(2),
+                        rs.getString(3),
+                        rs.getString(4));
+                team1.setId(rs.getInt(1));
+
+                Team team2 = new Team(
+                        rs.getString(6),
+                        rs.getString(7),
+                        rs.getString(8));
+                team2.setId(rs.getInt(5));
+                games.add(new Game(
+                        team1,
+                        team2,
+                        rs.getString(9),
+                        rs.getString(10)));
+            }
+        }
+        objects.add(games);
+        return objects;
+    }
+
+    @SneakyThrows
+    @Override
+    public List<Object> createTournament(String name, List<Team> teams) {
+        PreparedStatement preparedStatement = connection
+                .prepareStatement(
+                        "INSERT INTO TOURNAMENT (NAME) VALUES (?)",
+                        Statement.RETURN_GENERATED_KEYS);
+        preparedStatement.setString(1, name);
+        preparedStatement.executeUpdate();
+
+        int j;
+        try (ResultSet resultSet = preparedStatement.getGeneratedKeys()) {
+            resultSet.next();
+            j = resultSet.getInt(1);
+        }
+        PreparedStatement preparedStatement1 = connection
+                .prepareStatement(
+                        "INSERT INTO TEAMS (NAME, CAPTAIN, COACH) VALUES(?, ?, ?)",
+                        Statement.RETURN_GENERATED_KEYS);
+        tournamentService.register(teams);
+        for (Team t : teams) {
+            preparedStatement1.setString(1, t.getName());
+            preparedStatement1.setString(2, t.getCaptain());
+            preparedStatement1.setString(3, t.getCoach());
+            preparedStatement1.executeUpdate();
+            try (ResultSet resultSet1 = preparedStatement1.getGeneratedKeys()) {
+                resultSet1.next();
+                t.setId(resultSet1.getInt(1));
+            }
+        }
+        List<Game> games = tournamentService.start(teams);
+        PreparedStatement preparedStatement3 = connection
+                .prepareStatement(
+                        "INSERT INTO GAMES " +
+                                "(FIRST_TEAM, SECOND_TEAM, ROUND, RESULT, TOURNAMENT) " +
+                                "VALUES (?, ?, ?, ?, ?)");
+        for (Game game : games) {
+            preparedStatement3.setInt(1, game.getTeamFirst().getId());
+            preparedStatement3.setInt(2, game.getTeamSecond().getId());
+            preparedStatement3.setString(3, game.getRound());
+            preparedStatement3.setString(4, game.getResult());
+            preparedStatement3.setInt(5, j);
+            preparedStatement3.executeUpdate();
+
+        }
+        preparedStatement3.close();
+        return getTournament(j);
+
+    }
+
+    @SneakyThrows
+    private ArrayList<Team> createListTeams(ResultSet resultSet) {
+        String name = "", captain = "", coach = "";
```

Comment:
Avoid single line definition.
If you need to modify something it is harder to find what you need.